<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>426</width>
    <height>359</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="frame_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>419</width>
      <height>311</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QWidget" name="verticalLayoutWidget_2">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>40</y>
       <width>171</width>
       <height>191</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLabel" name="out_voltage_label">
        <property name="text">
         <string>Output voltage</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLCDNumber" name="out_voltage_lcdNumber"/>
      </item>
      <item>
       <widget class="QLabel" name="out_current_label">
        <property name="text">
         <string>Output current</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLCDNumber" name="out_current_lcdNumber"/>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="verticalLayoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>181</width>
       <height>191</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="set_voltage_label">
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDoubleSpinBox" name="set_voltage_DoubleSpinBox">
        <property name="buttonSymbols">
         <enum>QAbstractSpinBox::UpDownArrows</enum>
        </property>
        <property name="keyboardTracking">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="set_current_label">
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDoubleSpinBox" name="set_current_DoubleSpinBox">
        <property name="buttonSymbols">
         <enum>QAbstractSpinBox::UpDownArrows</enum>
        </property>
        <property name="keyboardTracking">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLCDNumber" name="Callibration_lcdNumber"/>
      </item>
     </layout>
    </widget>
    <widget class="QLabel" name="regim_label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>55</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <widget class="QPushButton" name="set_pushButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>250</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>Update</string>
     </property>
    </widget>
    <widget class="QPushButton" name="reset_pushButton">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>250</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>Reset</string>
     </property>
    </widget>
    <widget class="QPushButton" name="close_pushButton">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>250</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>Close</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>426</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
